warp/handlers/stats.go

```diff
@@ -1,7 +1,7 @@
 // (c) 2023, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package stats
```

Comment:
wondering if we should keep the packaging to `stats` for consistency with the sync stats.